libs/platform/unit/platform_unit_test.cpp

```diff
@@ -371,3 +375,82 @@ TEST_CASE("serialize_map_test", "[platform]")
     if (buffer != nullptr)
         free(buffer);
 }
+
+TEST_CASE("serialize_program_information_test", "[platform]")
+{
+    _test_helper test_helper;
+
+    ebpf_helper_function_prototype_t helper_prototype[] = {
+        {1000,
+         "helper_0",
+         EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+         {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
+        {1001,
+         "helper_1",
+         EBPF_RETURN_TYPE_INTEGER,
+         {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_VALUE}}};
+    ebpf_context_descriptor_t context_descriptor = {32, 0, 8, -1};
+    GUID PROGRAM_TYPE_TEST = {0x7ebe418c, 0x76dd, 0x4c2c, {0x99, 0xbc, 0x5c, 0x48, 0xa2, 0x30, 0x4b, 0x90}};
```

Comment:
```suggestion
    GUID PROGRAM_TYPE_TEST = {0x7ebe418c, 0x76dd, 0x4c2c, {0x99, 0xbc, 0x5c, 0x48, 0xa2, 0x30, 0x4b, 0x90}};
```
```suggestion
    GUID program_type_test = {0x7ebe418c, 0x76dd, 0x4c2c, {0x99, 0xbc, 0x5c, 0x48, 0xa2, 0x30, 0x4b, 0x90}};
```
ALL_CAPS is for defines/macros/enum values